src/screens/DataApi/DataCoverageTable.style.tsx

```diff
@@ -0,0 +1,77 @@
+import { TableHead, Typography } from '@material-ui/core';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+
+import { InfoIcon } from 'components/InfoTooltip/Tooltip.style';
+import styled from 'styled-components';
+
+export const TextWithTooltipContainer = styled.div`
+  ${InfoIcon} {
+    margin-left: 0.2rem;
+    vertical-align: middle;
+  }
+`;
+
+export const UnderlinedText = styled(Typography)`
+  text-decoration-line: underline;
+  text-decoration-style: dotted;
+  text-underline-position: under;
+  vertical-align: middle;
+  font-size: 14px;
+`;
+
+export const StyledTableHead = styled(TableHead)`
+  font-family: Roboto;
+  font-weight: 500;
+  font-size: 13px;
+  text-transform: uppercase;
+  background-color: white;
+`;
```

Comment:
We are trying to keep the minimum text size in the site to be 16px for accessibility - The text doesn't look super small to me, but maybe check how it looks at 16px?